source/draw/DrawState.ooc

```diff
@@ -82,5 +106,10 @@ DrawState: cover {
 	}
 	// Normalized transform
 	getTransformNormalized: func -> FloatTransform3D { this _transformNormalized }
-	draw: func { this target canvas draw(this) }
+	draw: func {
+		version(safe)
+			if (this target == null)
+				raise("Can't draw without a selected target.")
```

Comment:
Hmm could we introduce `raise: static func (condition: Bool, message: String) { if (condition) raise(message) }` ?
I think it could look better
```ooc
version(safe)
    raise(this target == null, "Null target")
```
@marcusnaslund what do you think ?